src/components/ExtendedReminder/index.tsx

```diff
@@ -127,19 +128,25 @@ const ExtendedReminder: FunctionComponent<ExtendedReminderProps> = ({
   } as const
 
   return (
-    <StyledContainer variant={variant} {...props}>
+    <StyledContainer
+      variant={variant}
+      aria-label={variant}
+      role="region"
+      {...props}
+    >
       <StyledBadgeContainer>
-        <Badge size="small" variant={badgeVariant[variant]}>
+        <Badge size="small" variant={badgeVariant[variant]} as="strong">
           <Icon
             mr="4px"
             color={theme.colorsDeprecated.white}
             name={icon}
             size={16}
+            aria-hidden="true"
           />
           {badgeText}
         </Badge>
       </StyledBadgeContainer>
-      <StyledTitle color={variant} mb={1} variant="bodyC">
+      <StyledTitle color={variant} mb={1} variant="bodyC" as="em">
```

Comment:
At this point that's not even a Typography :)

Reply:
Same as above, agree that it's strange

Reply:
also good for me: semantic is augmented. Unlike the strong tag upper, here we want to give more "sound" on the sentence to pay attention on (think of `em` as an old theater actor who declaims his text in an emphatic way)

You need to **adapt CSS** styles if needed because of defaults browsers styles (that are not semantic)